release_notes/ocp_3_5_release_notes.adoc

```diff
@@ -0,0 +1,1683 @@
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1233511[*BZ#1233511*])
+
+* In the web console, environment variables in the build file are no longer
+truncated after the `=` character. Previously, the environment variable values
+that contained an `=` character were being truncated.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1357107[*BZ#1357107*])
+
+* In the pod metrics page, the donut chart for current usage now appears to the
+right of the metrics sparkline. The new position allows you to see more metric
+data on the screen. Previously the donut chart was above the sparkline.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1387286[*BZ#1387286*])
+
+* Previously, some changes to a health check command or deployment hook command in
+the web console would not be saved. This happened when editing an existing
+command and adding or removing a single argument. The web console has been fixed
+to correctly save all edits to health check and deployment hook commands.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1411258[*BZ#1411258*])
+
+* Previously, you had to enter weights between `0` and `256` as integer values.
+When creating or editing routes that send traffic to two services in the web
+console, you can now specify the service weights as percentages using a slider
+control. You can still enter integer weights if desired.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1416882[*BZ#1416882*])
+
+* On the *Add to Project* page of the web console, if you entered an invalid
+setting for some advanced options, then hide the advanced options, the form
+would be submitted with invalid values, causing errors when creating some
+resources like horizontal pod autoscalers. The web console has been changed to
+correctly validate these fields so that you can not submit the form with invalid
+values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1419887[*BZ#1419887*])
+
+* Previously, the link to download the OpenShift CLI linked to the incorrect
+version (of OpenShift Origin). The link has been updated, and the link downloads
+the correct version.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1421949[*BZ#1421949*])
+
+* Previously, the *Create a Secret* and *Add Config Files* buttons when creating
+using the web console mistakenly linked to the other page. The buttons have been
+corrected.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1425728[*BZ#1425728*])
+
+* Dates now use the word form over the number form to avoid ambiguity (For example,
+May 4, 2016 instead of 05/04/2017).
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1333101[*BZ#1333101*])
+
+* Previously, dropdown menus on the web console overlay the navigation menu
+dropdowns, blocking the view and usability of the navigation menu dropdowns. The
+navigation menu dropdown’s z-index has been set to a value greater than that of
+page content dropdowns, resulting in navigation menu dropdowns to always appear
+on top of page content dropdowns.
+(l
```

Comment:
@mpbarrett Is there anything that should be listed in the Known Issues section? I did not have anything in my notes. Thanks!